.github/workflows/pull-request.yml

```diff
@@ -53,12 +53,12 @@ jobs:
       - name: Checkout forkliftci
         uses: actions/checkout@v4
         with:
-          repository: kubev2v/forkliftci
+          repository: bennyz/forkliftci
```

Comment:
that would be excellent to have also an image-based vm in our CI

Reply:
unfortunately the current code isn't too friendly towards adding an additional vm, but this code path should be tested in e2e